src/pivot-table/data/helpers/get-random-uuid.ts

```diff
@@ -0,0 +1,5 @@
+import { v4 as uuidv4 } from "uuid";
+
+const getRandomUUID = () => crypto?.randomUUID?.() ?? uuidv4();
```

Comment:
crypto seems like having a good support for [most browsers](https://developer.mozilla.org/en-US/docs/Web/API/Crypto#browser_compatibility), why should we still have uuidv4 as fallback solution?

Reply:
`randomUUID` is not available for all browsers in non-secure contexts https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID